test/spec.js

```diff
@@ -28,4 +29,42 @@ describe('runner', function() {
     });
   });
 
+  function testWithBlaster(numLines) {
+    it(`blaster ${numLines}`, function () {
+      if (numLines >= 20000) {
+        this.timeout(100 * numLines); // A rough heuristic that works on a MacBook with plenty of headroom
+      }
+      let chunks = [];
+      function output(data) {
+        chunks.push(data.toString());
+      }
+      const numLinesAsString = `${numLines}`;
+      const executable = './testers/blaster.js';
+      const args = [numLinesAsString];
+      return runner.run({executable, args, output})
```

Comment:
Running it this way might not introduce realistic delays that you would see when redirecting to a file, especially via NFS, but I'm not sure how to improve the test, other than to write to an actual file.